### PR TITLE
es6: classes

### PR DESCRIPTION
**Pull request by ikokostya**
Relevant to https://github.com/yandex/codestyle/issues/33

### CODE REVIEW

**Comment by dmikis** [comment]
OK

File: javascript.ru.md [hunk 1]
```diff
@@ -576,9 +578,131 @@ var FooClass = inherit({
...
+    this.y = y;
+    this.radius = radius;
+}
+
+Circle.prototype.area = function () {
+    return Math.PI * this.radius * this.radius;
+};
+```
+
+* После имени класса ставится 1 пробел:
```
Comment (tarmolov) [inline javascript.ru.md]: Цифру можно опустить; "пробел" уже означает, что он один.

**Comment by tarmolov** [comment]
:ok:

File: javascript.md [hunk 2]
```diff
@@ -566,6 +568,128 @@ var FooClass = inherit({
 });
 ```
 
+## ECMAScript 6
+
+### Classes
+
+* For class definition `class` keyword should be used:
```
Comment (dodev) [inline javascript.md]: .. the `class` keyword

File: javascript.md [hunk 3]
```diff
@@ -566,6 +568,128 @@ var FooClass = inherit({
...
+    this.y = y;
+    this.radius = radius;
+}
+
+Circle.prototype.area = function () {
+    return Math.PI * this.radius * this.radius;
+};
+```
+
+* There should be one whitespace after class name:
```
Comment (dodev) [inline javascript.md]: ... after the class name:

File: javascript.md [hunk 4]
```diff
@@ -566,6 +568,128 @@ var FooClass = inherit({
...
+
+**Bad:**
+
+```js
+class Circle {
+    area () {}
+}
+```
+
+* There should be one whitespace before opening curly brace of method body:
```
Comment (dodev) [inline javascript.md]: ... before *the* opening curly brace of *method's* body:

File: javascript.md [hunk 5]
```diff
@@ -566,6 +568,128 @@ var FooClass = inherit({
...
+
+**Bad:**
+
+```js
+class Circle {
+    area(){}
+}
+```
+
+* Constructor (if exists) should be first method in class:
```
Comment (dodev) [inline javascript.md]: The constructor (if exists) should be the first method in a class definition:

File: javascript.md [hunk 6]
```diff
@@ -566,6 +568,128 @@ var FooClass = inherit({
...
+
+```js
+class Circle {
+    area() {}
+
+    constructor() {}
+}
+```
+
+* For inheritance `extends` keyword should be used:
```
Comment (dodev) [inline javascript.md]: For inheritance the `extends` keyword...

**Comment by dodev** [comment]
:+1:

**Comment by ikokostya** [comment]
@dodev thanks

**Comment by twoRoger** [comment]
:metal:

**Pull request merged**